home.html

```diff
@@ -0,0 +1,133 @@
+                    <li><a href="#" class="hover-link"><i class="fa-solid fa-user"></i>&nbsp;&nbsp;&nbsp;Contact Sales</a></li>
+                </ul>
+            </div>
+            <div class="top-banner-right">
+                <ul class="flex">
+                    <li><a href="#" class="hover-link"><i class="fa-solid fa-light fa-magnifying-glass"></i></a></li>
+                    <li><a href="#" class="hover-link">Log in</a></li>
+                    <li><a href="#" class="hover-link">Customer Support</a></li>
+                    <li><a href="#" class="hover-link">About&nbsp;&nbsp;&nbsp;<i class="fa-solid fa-caret-down"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </div>
+
+    <!-- Nav Bar -->
+
+    <nav>
+        <div class="container main-nav flex">
+            <a href="#" class="nav-logo-toggle hover-link">
+                <i class="fa-brands fa-hubspot fa-2xl" style="color: #ff9300;"></i>
+            </a>
+            <div class="nav-left flex">
+                <a href="#" class="company-logo">
+                    <img src="hubspotNameImg.png" alt="company logo">
+                </a>
+                <ul class="flex">
+                    <li><a href="#" class="hover-link">Software&nbsp;&nbsp;&nbsp;<i class="fa-solid fa-caret-down"></i></a></li>
+                    <li><a href="#" class="hover-link">Pricing</a></li>
+                    <li><a href="#" class="hover-link">Resources&nbsp;&nbsp;&nbsp;<i class="fa-solid fa-caret-down"></i></a></li>
+                </ul>
+            </div>
+            <div class="nav-right flex">
+                <ul class="flex">
+                    <li><a href="#" class="hover-link secondary-button">Get a demo</a></li>
+                    <li><a href="#" class="hover-link primary-button">Get started free</a></li>
+                </ul>
+            </div>
+            <a href="#" class="nav-toggle hover-link">
+                <i class="fa-solid fa-bars fa-2xl"></i>
+            </a>
+        </div>
+    </nav>
+
+    <!-- Header Section -->
+
+    <header>
+        <div class="container header-section flex">
+            <div class="header-left">
+                <h1>Grow better with Hubspot</h1>
+                <p>Software that's powerful, not overpowering. Seamlessly connect your data, teams, and customers on one CRM platform that grows with your business.</p>
+                <a href="#" class="hover-link secondary-button">Get a demo</a>
+                <a href="#" class="hover-link primary-button">Get started free</a>
+                <p>Get a demo of our premium software, or get started with free tools.</p>
+            </div>
+            <div class="header-right">
+                <img src="./homeimgfinal.png" alt="hero image">
+            </div>
+        </div>
+    </header>
+
+    <!-- Customers Section -->
+
+    <section class="customers">
+        <div class="container flex customers-section">
```

Comment:
i think you didn't undertand flex properly, you need to study it properly and redo this and above one.